docs/index.md

```
@@ -30,53 +40,83 @@ Cliffy is a powerful CLI framework that simplifies the creation of command-line
+        print(f"🎉 Marked task {id} as complete")
     ```
 
+    ![todo-demo](images/demo.gif)
 
-- [Getting Started](getting-started.md)
-- [Features](features.md)
-- [Schema](schema.md)
-- [Examples](https://github.com/jaykv/cliffy/tree/main/examples)
+For more examples, check [examples](examples/) directory.
 
+### Why cliffy
```

Markdownlint (0.37.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)